src/HydraAuction/Delegate/Interface.hs

```diff
@@ -0,0 +1,35 @@
+module HydraAuction.Delegate.Interface (
+  FrontendKind (..),
+  KnownFrontendRequest (..),
+  DelegateError (..),
+  DelegateResponse (..),
+  FrontendRequest (..),
+) where
+
+-- Prelude imports
+import Prelude
+
+-- Cardano imports
+import Cardano.Api.UTxO (UTxO)
+
+-- HydraAuction imports
+import HydraAuction.Types (AuctionTerms (..), Natural)
+
+data FrontendKind = Seller | Bidder
+data KnownFrontendRequest
```

Comment:
We should identify the actor that is making the request as well

Reply:
Why? Seller only could be the one actor. Bidders provide their signatures on with newBid command.

Will think of that further.

Reply:
Yep, that is in Milestone 6. It is separated to issue #203 . Actually I wanted to discuss if you wanna take this issue next, cuz seems that M4.2 work will be splitted between me and @mangas